src/editor/editor.js

```diff
@@ -310,7 +297,8 @@ export default class Editor {
 			resolve(
 				editor.initPlugins()
 					.then( () => {
-						editor.fire( 'dataReady' );
+						// Fire `data#ready` event manually as `data#init()` method is not used.
+						editor.data.fire( 'ready' );
```

Comment:
I think the whole `Editor#create` method should be removed since it is meaningless.

Reply:
It was removed in https://github.com/ckeditor/ckeditor5-core/pull/157/commits/60d06c394c8aeeb7d3bb7cb138e67b24d529fede. Unless you meant some other place in the code (it's only used in tests now)?